application/config/config.go

```diff
@@ -872,3 +875,16 @@ func (c *Config) IdeName() string {
 func (c *Config) IsFedramp() bool {
 	return c.Engine().GetConfiguration().GetBool(configuration.IS_FEDRAMP)
 }
+
+func (c *Config) IsAnalyticsPermitted() bool {
+	u, err := url.Parse(c.Engine().GetConfiguration().GetString(configuration.API_URL))
```

Comment:
Initialization order may be interesting here - is the endpoint already set when IsAnalyticsPermitted is called?

Reply:
We went through the server's `initlialize` handler, and made sure that all calls to `IsAnalyticsPermitted` happen after the API URL has been set in the configuration 👍  `updateTelemetry` is the first function inside the `initialize` handler that is checking if analytics are permitted, and it's being called after setting the API URL. Further calls are done in the `initialized` handler, which also happens afterwards.